shimmingtoolbox/load_nifti.py

```diff
@@ -0,0 +1,168 @@
+#!usr/bin/env python3
+# -*- coding: utf-8
+
+import os
+import logging
+import numpy as np
+import nibabel as nib
+import json
+import math
+
+logger = logging.getLogger(__name__)
+PHASE_SCALING_SIEMENS = 4096
+
+
+def load_nifti(path_data, modality='phase'):
+    """
+    Load data from a NIFTI type file with nibabel.
```

Comment:
i find it confusing: here it says the function loads from a nifti file, but `path_data` is expecting a folder (not a file)

Reply:
Yes it should say: "from a directory containing NIFTI type files"